src/content/blog/astro-4-beta.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Astro 4.0 Beta"
+description: "Astro 4.0 beta is released."
+publishDate: "November 22, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-4beta/post-header-4.0beta.webp"
+socialImage: "/src/content/blog/_images/astro-4beta/post-header-4.0beta.webp"
+lang: "en"
+---
+
+Today we've released the first beta of Astro 4! Those feeling adventurous can install the new beta, review our in-progress upgrade docs, and report any bugs you find to the [Astro discord](https://astro.build/chat).
+
+## What to expect in Astro 4
+
+We're releasing Astro 4 to stay up to date with the latest version of Vite, which just released Vite 5 last month.
```

Comment:
```suggestion
We are releasing Astro 4 to stay up to date with [Vite](https://vitejs.dev/), our internal build tool. Astro 4 includes the latest version of [Vite 5](https://vitejs.dev/blog/announcing-vite5), which was just released last month.
```

You could maybe trim this down a bit more from my suggestion! The main suggestion here is to quickly mention what Vite is and what this means to the Astro user/reader (aka, "so what?").